src/include/main/client_context.h

```diff
@@ -123,6 +123,10 @@ class KUZU_API ClientContext {
     void setDefaultDatabase(AttachedKuzuDatabase* defaultDatabase_);
     bool hasDefaultDatabase();
 
+    void runFuncInTransaction(const std::function<void(void)>& fun);
```

Comment:
why did you move to public?

Reply:
This was originally used to register the json type, but now it's not necessary